sirepo/package_data/static/js/viz3d.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var srlog = SIREPO.srlog;
+var srdbg = SIREPO.srdbg;
+
+SIREPO.app.config(function() {
+    SIREPO.appFieldEditors += ``;
+});
+
+SIREPO.app.factory('viz3dService', function(appState) {
+    const self = {};
+
+    self.computeModel = () => 'viz3dReport';
+
+    appState.setAppService(self);
+    return self;
+});
+
+SIREPO.app.controller('Viz3dSourceController', function (appState, panelState, persistentSimulation, viz3dService, $scope) {
+    const self = this;
+    self.simScope = $scope;
+
+    self.simHandleStatus = data => {};
+    self.simState = persistentSimulation.initSimulationState(self);
+});
+
+
+
+SIREPO.app.directive('appFooter', function() {
+    return {
+        restrict: 'A',
+        scope: {
+            nav: '=appFooter',
+        },
+        template: `
+            <div data-common-footer="nav"></div>
+        `,
+    };
+});
+
+
+SIREPO.app.directive('appHeader', function(appState, panelState) {
+    return {
+        restrict: 'A',
+        scope: {
+            nav: '=appHeader',
+        },
+        template: `
+            <div data-app-header-brand="nav"></div>
+            <div data-app-header-left="nav"></div>
+            <div data-app-header-right="nav">
+              <app-header-right-sim-loaded>
+                <div data-sim-sections="">
+                  <li class="sim-section" data-ng-class="{active: nav.isActive(\'source\')}"><a href data-ng-click="nav.openSection(\'source\')"><span class="glyphicon glyphicon-flash"></span> Source</a></li>
```

Comment:
Name this "Visualization" instead of "Source" and then fix up the name accordingly elsewhere (ex "Viz3dSourceController as source" -> "Viz3dVisualizationController as visualization")